Client/Frontend/WebExtensions/WebExtension.swift

```diff
@@ -0,0 +1,253 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import SwiftyJSON
+import WebKit
+import ZipArchive
+
+fileprivate let ManifestVersion = 2
+
+fileprivate let WindowProxyJS: String = {
+    let path = Bundle.main.path(forResource: "WindowProxy", ofType: "js")!
+    return try! NSString(contentsOfFile: path, encoding: String.Encoding.utf8.rawValue) as String
+}()
```

Comment:
nit; in case we want to quickly change loading (maybe for debugging, or sentry logging):
```
fileprivate func loadJS(_ name: String) -> String { ... } 
let WindowProxyJS = loadJS("WindowProxy")
let BrowserEventJS = loadJS("BrowserEventJS")
```